slides.tex

```diff
@@ -153,10 +171,19 @@
         \end{figure}
     \end{frame}
 
+    \begin{frame}
```

Comment:
Move this after data lake use case slide

Reply:
Slides updated.